rdb/project.clj

```diff
@@ -0,0 +1,17 @@
+(defproject mongodb "0.1.0-SNAPSHOT"
+  :description "RethinkDB Jepsen Tests"
+;  :url "http://github.com/rethinkdb/jepsen"
```

Comment:
You can use the jepsen repo URL if you like, but it's not mandatory. I'm more concerned that the project name is "mongodb". I suggest "jepsen.rethinkdb".